data/donations.json

```diff
@@ -222,5 +222,277 @@
         "sum": 25,
         "userId": "74e1510f-46d4-4d53-a4ac-98a70703c9b0",
         "ideaId": "e4a253af-d5b9-4384-9f4b-868d99205915"
+    },
+    {
+        "id": "c458ff39-83ae-46bf-a66a-d5b5728ff30b",
+        "firstName": "Anonymous",
+        "sum": null,
```

Comment:
How is it possible to have a donation with `null` sum?

Reply:
![Screenshot_20](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/a29cb246-2fde-42e2-8af0-53a745d6a4e0)
That works pretty fine. Maybe before I've implemented validations, such result occurred.

Reply:
So this data is corrupt. Can you remove donations with `null` sum please

Reply:
Fixed